cmake/FindProtobufC.cmake

```diff
@@ -90,17 +90,21 @@ find_library(PROTOBUFC_LIBRARY
 mark_as_advanced(PROTOBUFC_LIBRARY)
 
 find_path(PROTOBUFC_INCLUDE_DIR
-							NAMES "google/protobuf-c/protobuf-c.h"
+							NAMES "protobuf-c.h" "google/protobuf-c/protobuf-c.h"
 							PATHS "/usr" "/usr/local" "/opt" ENV PROTOBUFC_ROOTDIR
 							PATH_SUFFIXES "include")
 mark_as_advanced(PROTOBUFC_INCLUDE_DIR)
 
 find_program(PROTOBUFC_COMPILER
-							NAMES "protoc-c"
+							NAMES "protoc" "protoc-c"
 							PATHS "/usr" "/usr/local" "/opt" ENV PROTOBUFC_ROOTDIR
 							PATH_SUFFIXES "bin")
 mark_as_advanced(PROTOBUFC_COMPILER)
 
+IF (DEFINED PROTOBUF_PROTOC_EXECUTABLE)
```

Comment:
so PROTOBUF_PROTOC_EXECUTABLE is provided by the user ? Is it for an alternative name than protoc or protoc-c, or for probing a different location? If the later, doesn't conda-forge setting of CMAKE_PREFIX_PATH or other CMake variables enough to find it?

Reply:
For conda-forge builds, we are cross-compiling on OSX, and we have to explicitly provide the host's `protoc` instead of letting it pick up the build's `protoc`. I needed a way to specify it or the wrong arch gets picked up and it errors.

Reply:
As far as I can see, cmake -DPROTOBUFC_COMPILER=/path/to/proto-c should work without this patch, given that the find_program() output variable is a cache variable